packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -4,16 +4,17 @@
 import classnames from 'classnames';
 import { first, last, omit } from 'lodash';
 import { animated } from 'react-spring/web.cjs';
+import mergeRefs from 'react-merge-refs';
```

Comment:
This was technically not needed, but I believe this is better than the "fallback" we had previously. The previous fallback wouldn't have worked with functions as refs...

Reply:
Why do we need to support functions?

Reply:
we can just say we don't, but I think a React  component user would expect  any ref to work not just the ones  created  with  createRef.

`<Block.div ref={ node => dosomething } />`

Also, this doesn't have any downside (including bundle size)

Reply:
Ok, sounds good